styles/globals.css

```diff
@@ -103,22 +103,27 @@ header .links {
   display: flex;
   justify-content: flex-end;
   position: relative;
+  width: fit-content;
+  margin: 0 auto;
 }
 
 .clear-icon {
   position: absolute;
-  right: 0.8rem;
-  top: 0.3rem;
+  top: 25px;
+  font-size: 20px;
+  right: 10px;
   cursor: pointer;
 }
 
 .searchbox {
   font-size: 18px;
-  padding: 10px;
-  margin: 5px;
+  padding: 20px;
+  text-align: center;
+  margin: 20px auto;
   background: rgb(234, 239, 241);
   border: 1px solid rgb(47, 47, 47);
   border-radius: 3px;
+  width: 30vw !important;
```

Comment:
Why do we need !important?

Reply:
@omriattiya One thong u may want to do is, `width: fit-content;` and make the textbox width 80%

Reply:
I removed the `important`.

Reply:
> @omriattiya One thing u may want to do is, remove `width: fit-content;` and make the textbox width 80%

that will destroy the UI and move the clear button far away. It's better to make it fit content so the clear button is always relative to it. If you want to change the width of the whole searchbox you need to do it in `searchbox` not in `search-box`

Reply:
@omriattiya I get your point. You are right in handling the cross. I am a bit concerned about the search box is very small on phone. Any idea on fixing it?

![image](https://user-images.githubusercontent.com/3633137/135630233-06a56c7e-74ce-42a2-b112-fdb6bc3f7b03.png)

Reply:
added a `min-width` . That should fix it.